docs/integrations/sources/sharetribe.md

```diff
@@ -0,0 +1,53 @@
+# Sharetribe
+
+This is the setup guide for the Sharetribe source connector which ingests data from the sharetribe integrations API.
+
+Sharetribe is a no code marketplace builder tool. The source supports a number of API changes.
+For more information, checkout the website https://www.sharetribe.com/
+
+This source uses the OAuth configuration for handling requests.
+A client_ID and client_secret is required along with a refresh_token in order to setup a connection.
+You will need to create an application to obtain the client credientials.
+For more details about the API, check out https://www.sharetribe.com/api-reference/integration.html
+
+## Set up the Adjust source connector
+
+1. Click **Sources** and then click **+ New source**.
+2. On the Set up the source page, select **Sharetribe** from the Source type dropdown.
+3. Enter a name for your new source.
+4. For **Client Id**, enter your client_id obtained in the previous step.
+5. For **Client Secret**, enter your client_secret obtained in the previous step.
+6. For **Refresh Token**, enter your refresh_token obtained in the previous step.
+7. For **Access Token**, enter your access token obtained in the previous step.
+8. For **Start Date**, enter a start date from where you would like to sync the records.
+9. Click **Set up source**.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id` | `string` | Client ID.  |  |
+| `client_secret` | `string` | Client secret.  |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `oauth_access_token` | `string` | Access token. The current access token. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+| `oauth_token_expiry_date` | `string` | Token expiry date. The date the current access token expires in. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| users | id | Default pagination | ✅ |   ✅ |
+| marketplace | id | No pagination | ✅ |  ❌  |
+| listings | id | Default pagination | ✅ |  ✅  |
+| transactions | id | Default pagination | ✅ |  ✅  |
+| events | id | Default pagination | ✅ |  ✅  |
```

Comment:
Looking into the Sharetribe website `bookings` it is an important endpoint missing in your connector.
Please include this endpoint, after let's ship it.
<img width="630" alt="image" src="https://github.com/user-attachments/assets/a7e525d2-48d1-4b3e-8850-b0080cbf21c2">

Reply:
The same for `reviews` and `messages`.

Reply:
Done!